Source/OxyPlot/Series/HeatMapSeries.cs

```diff
@@ -122,6 +122,21 @@ public HeatMapSeries()
         public bool Interpolate { get; set; }
 
         /// <summary>
+        /// Gets or sets a value indicating whether to use discrete rectangles when rendering (ignored if Interpolate is set to true). The default value is <c>false</c>.
+        /// </summary>
+        public bool RenderDiscreteRectangles { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to treat the x dimension as logarithmic. The default value is <c>false</c>.
+        /// </summary>
+        public bool LogX { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to treat the y dimension as logarithmic. The default value is <c>false</c>.
+        /// </summary>
+        public bool LogY { get; set; }
```

Comment:
I understand that these properties say something about the data. 
E.g. if you have a log axis from 1 to 100 and you have heat map data with 3 points, then the `LogX` and `LogY` properties tell if the data should be given at positions (1, 10, 100) or (1, 50.5, 100). Is that right?